internal/engine/upper_test.go

```diff
@@ -33,6 +33,13 @@ import (
 	"github.com/windmilleng/tilt/internal/watch"
 )
 
+const (
+	testTiltfile = `def foobar():
```

Comment:
maybe a slightly more descriptive name?
```suggestion
	testSimpleTiltfile = `def foobar():
```

Reply:
I think it'd be cool to have a 'lil library of these: multiple manifests, mounts, runs, etc.

Reply:
i agree! but also the way we load manifests is inconsistent in this file -- some we `load` from the tiltfile, and some we assemble piecemeal, and urgh.